src/util/prolog/prologResult.ts

```diff
@@ -0,0 +1,120 @@
+import * as ld from "lodash";
```

Comment:
More conventionally, `lodash` is imported as `_`. But I prefer explicitly importing only those methods that you really use.